lightningd/peer_control.c

```diff
@@ -718,6 +718,21 @@ static void json_add_peer(struct lightningd *ld,
 			      &channel->funding_txid);
 		json_add_bool(response, "private",
 				!(channel->channel_flags & CHANNEL_FLAGS_ANNOUNCE_CHANNEL));
+
+		// FIXME @conscott : Modify this when dual-funded channels
+		// are implemented
+		json_object_start(response, "funding_allocation_msat");
+		if (channel->funder == LOCAL) {
+			json_add_u64(response, pubkey_to_hexstr(tmpctx, &p->id), 0);
+			json_add_u64(response, pubkey_to_hexstr(tmpctx, &ld->id),
+					channel->funding_satoshi * 1000);
```

Comment:
What's the rationale for multiplying this by 1000? `funding_satoshi` should be a satoshi value already.